errortypes/code.go

```diff
@@ -0,0 +1,29 @@
+package errortypes
+
+// Defines numeric codes for well-known errors in this package.
+const (
+	NoErrorCode = iota
+	TimeoutCode
+	BadInputCode
+	BlacklistedAppCode
+	BadServerResponseCode
+	FailedToRequestBidsCode
+	BidderTemporarilyDisabledCode
+	BlacklistedAcctCode
+	AcctRequiredCode
+	InvalidPrivacyConsentCode
```

Comment:
`InvalidPrivacyConsentCode` was formerly the `WarningCode`. I think it's safe to remove since the `WarningCode` was never documented or written out in the response. Do you think it's ok to continue counting here and mix error and warning codes? Or maybe add, say, a 10,000 gap between errors and warnings? Or keep them entirely separate?

Reply:
I lean towards the gap idea to keep things simple and organized, but I am flexible on this idea.

Reply:
I like that too. I added a gap. Error codes start at 1. Warning codes start at 10,001 (or would this better be 1,001?).